src/main/java/org/takes/rq/RqHeaders.java

```diff
@@ -90,35 +89,19 @@ public Base(final Request req) {
             super(req);
         }
         @Override
-        public Iterable<String> header(final CharSequence key)
+        public List<String> header(final CharSequence key)
             throws IOException {
             final Map<String, List<String>> map = this.map();
             final List<String> values = map.get(
                 key.toString().toLowerCase(Locale.ENGLISH)
             );
-            final Iterable<String> iter;
             if (values == null) {
-                iter = new VerboseIterable<String>(
-                    Collections.<String>emptyList(),
-                    new Sprintf(
-                        // @checkstyle LineLengthCheck (1 line)
-                        "there are no headers by name \"%s\" among %d others: %s",
-                        key, map.size(), map.keySet()
-                    )
-                );
-            } else {
-                iter = new VerboseIterable<String>(
-                    values,
-                    new Sprintf(
-                        "there are only %d headers by name \"%s\"",
-                        values.size(), key
-                    )
-                );
+                return Collections.emptyList();
             }
-            return iter;
+            return values;
```

Comment:
@aliceice please check out this article http://www.yegor256.com/2015/08/18/multiple-return-statements-in-oop.html